<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DashboardWindow</class>
 <widget class="QMainWindow" name="DashboardWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hotel Management System - Dashboard</string>
  </property>
  <property name="styleSheet">
   <string>QMainWindow { background-color: #f8f9fa; }</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="mainLayout">
    <item>
     <widget class="QWidget" name="sidebar" native="true">
      <property name="styleSheet">
       <string>QWidget { background-color: #2c3e50; }</string>
      </property>
      <layout class="QVBoxLayout" name="sidebarLayout">
       <item>
        <widget class="QLabel" name="logoLabel">
         <property name="styleSheet">
          <string>QLabel { color: white; font-size: 18px; font-weight: bold; padding: 20px; border-bottom: 1px solid #34495e; }</string>
         </property>
         <property name="text">
          <string>üè® HOTEL</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="dashboardBtn">
         <property name="styleSheet">
          <string>QPushButton { background-color: #34495e; color: white; border: none; text-align: left; padding-left: 20px; font-weight: bold; } QPushButton:hover { background-color: #3498db; }</string>
         </property>
         <property name="text">
          <string>üìä Dashboard</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="reservasiBtn">
         <property name="styleSheet">
          <string>QPushButton { background-color: #2c3e50; color: white; border: none; text-align: left; padding-left: 20px; } QPushButton:hover { background-color: #3498db; }</string>
         </property>
         <property name="text">
          <string>üìÖ Reservasi</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="kamarBtn">
         <property name="styleSheet">
          <string>QPushButton { background-color: #2c3e50; color: white; border: none; text-align: left; padding-left: 20px; } QPushButton:hover { background-color: #3498db; }</string>
         </property>
         <property name="text">
          <string>üö™ Kamar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pelangganBtn">
         <property name="styleSheet">
          <string>QPushButton { background-color: #2c3e50; color: white; border: none; text-align: left; padding-left: 20px; } QPushButton:hover { background-color: #3498db; }</string>
         </property>
         <property name="text">
          <string>üë• Pelanggan</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pembayaranBtn">
         <property name="styleSheet">
          <string>QPushButton { background-color: #2c3e50; color: white; border: none; text-align: left; padding-left: 20px; } QPushButton:hover { background-color: #3498db; }</string>
         </property>
         <property name="text">
          <string>üí≥ Pembayaran</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="laporanBtn">
         <property name="styleSheet">
          <string>QPushButton { background-color: #2c3e50; color: white; border: none; text-align: left; padding-left: 20px; } QPushButton:hover { background-color: #3498db; }</string>
         </property>
         <property name="text">
          <string>üìà Laporan</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>150</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="logoutBtn">
         <property name="styleSheet">
          <string>QPushButton { background-color: #e74c3c; color: white; border: none; text-align: left; padding-left: 20px; } QPushButton:hover { background-color: #c0392b; }</string>
         </property>
         <property name="text">
          <string>üö™ Logout</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="contentArea" native="true">
      <layout class="QVBoxLayout" name="contentLayout">
       <item>
        <widget class="QWidget" name="topBar" native="true">
         <property name="styleSheet">
          <string>QWidget { background-color: white; border-bottom: 1px solid #ecf0f1; }</string>
         </property>
         <layout class="QHBoxLayout" name="topBarLayout">
          <item>
           <widget class="QLabel" name="welcomeMsg">
            <property name="styleSheet">
             <string>QLabel { color: #2c3e50; font-size: 22px; font-weight: bold; margin-left: 30px; }</string>
            </property>
            <property name="text">
             <string>Welcome, Admin</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>400</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="dateTimeLabel">
            <property name="styleSheet">
             <string>QLabel { color: #7f8c8d; font-size: 12px; margin-right: 30px; }</string>
            </property>
            <property name="text">
             <string>17 Nov 2025</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="statsArea" native="true">
         <layout class="QHBoxLayout" name="statsLayout">
          <item>
           <widget class="QWidget" name="stat1" native="true">
            <property name="styleSheet">
             <string>QWidget { background-color: white; border-radius: 10px; border: 1px solid #ecf0f1; }</string>
            </property>
            <layout class="QVBoxLayout" name="stat1Layout">
             <item>
              <widget class="QLabel" name="stat1Title">
               <property name="styleSheet">
                <string>QLabel { color: #7f8c8d; font-size: 12px; font-weight: bold; }</string>
               </property>
               <property name="text">
                <string>Total Reservasi</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="stat1Value">
               <property name="styleSheet">
                <string>QLabel { color: #3498db; font-size: 32px; font-weight: bold; }</string>
               </property>
               <property name="text">
                <string>125</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="stat2" native="true">
            <property name="styleSheet">
             <string>QWidget { background-color: white; border-radius: 10px; border: 1px solid #ecf0f1; }</string>
            </property>
            <layout class="QVBoxLayout" name="stat2Layout">
             <item>
              <widget class="QLabel" name="stat2Title">
               <property name="styleSheet">
                <string>QLabel { color: #7f8c8d; font-size: 12px; font-weight: bold; }</string>
               </property>
               <property name="text">
                <string>Kamar Tersedia</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="stat2Value">
               <property name="styleSheet">
                <string>QLabel { color: #2ecc71; font-size: 32px; font-weight: bold; }</string>
               </property>
               <property name="text">
                <string>18</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="stat3" native="true">
            <property name="styleSheet">
             <string>QWidget { background-color: white; border-radius: 10px; border: 1px solid #ecf0f1; }</string>
            </property>
            <layout class="QVBoxLayout" name="stat3Layout">
             <item>
              <widget class="QLabel" name="stat3Title">
               <property name="styleSheet">
                <string>QLabel { color: #7f8c8d; font-size: 12px; font-weight: bold; }</string>
               </property>
               <property name="text">
                <string>Pendapatan Bulan Ini</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="stat3Value">
               <property name="styleSheet">
                <string>QLabel { color: #f39c12; font-size: 32px; font-weight: bold; }</string>
               </property>
               <property name="text">
                <string>Rp 45M</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="stat4" native="true">
            <property name="styleSheet">
             <string>QWidget { background-color: white; border-radius: 10px; border: 1px solid #ecf0f1; }</string>
            </property>
            <layout class="QVBoxLayout" name="stat4Layout">
             <item>
              <widget class="QLabel" name="stat4Title">
               <property name="styleSheet">
                <string>QLabel { color: #7f8c8d; font-size: 12px; font-weight: bold; }</string>
               </property>
               <property name="text">
                <string>Pelanggan Aktif</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="stat4Value">
               <property name="styleSheet">
                <string>QLabel { color: #e74c3c; font-size: 32px; font-weight: bold; }</string>
               </property>
               <property name="text">
                <string>89</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="chartsArea" native="true">
         <layout class="QHBoxLayout" name="chartsLayout">
          <item>
           <widget class="QWidget" name="chart1" native="true">
            <property name="styleSheet">
             <string>QWidget { background-color: white; border-radius: 10px; border: 1px solid #ecf0f1; }</string>
            </property>
            <layout class="QVBoxLayout" name="chart1Layout">
             <item>
              <widget class="QLabel" name="chart1Title">
               <property name="styleSheet">
                <string>QLabel { color: #2c3e50; font-size: 14px; font-weight: bold; padding: 15px; }</string>
               </property>
               <property name="text">
                <string>Grafik Occupancy Rate</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="chart1Content">
               <property name="styleSheet">
                <string>QLabel { color: #95a5a6; padding: 40px; }</string>
               </property>
               <property name="text">
                <string>[Chart Area - 85% Occupancy]</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="chart2" native="true">
            <property name="styleSheet">
             <string>QWidget { background-color: white; border-radius: 10px; border: 1px solid #ecf0f1; }</string>
            </property>
            <layout class="QVBoxLayout" name="chart2Layout">
             <item>
              <widget class="QLabel" name="chart2Title">
               <property name="styleSheet">
                <string>QLabel { color: #2c3e50; font-size: 14px; font-weight: bold; padding: 15px; }</string>
               </property>
               <property name="text">
                <string>Revenue Trend</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="chart2Content">
               <property name="styleSheet">
                <string>QLabel { color: #95a5a6; padding: 40px; }</string>
               </property>
               <property name="text">
                <string>[Chart Area - Monthly Revenue]</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
